### ISSUE TITLE
Rename files and enforce Presenters/View architecture.

### ISSUE THREAD

**Issue by DowLucas** [DowLucas/PubGo#12]
Currently, we dont name the files views or presenters but this is something that needs to be done.

**Comment by erikmartinessanches** [comment]
Indeed. Clicker.js should for example be split up into a ClickerPresenter and a ClickerView. 
In this example, I'd keep the `useDispatch` and `useSelector` calls in ClickerPresenter and pass `clickerValue`, `selectedEvent` `onIncrease` and `onDecrease` as props to ClickerView.

Remember that `useSelector` and `useDispatch` manipulate the application model and that should only be done from a presenter and not a view. Therefore the view-presenter separation is kind of important.
